index.html

```diff
@@ -0,0 +1,449 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <!--Mobile menu-->
+      <div class="menu-bars">
+        <i class="fas fa-bars open-btn"></i>
+        <i class="fas fa-times close-btn"></i>
```

Comment:
- [ ] Nice job implementing the menu close button. it works well as expected. nevertheless, we have one small issue of its position. please lets perfect that according to the original design.
- your design 👇🏾 ;
![Screenshot (7)](https://user-images.githubusercontent.com/65298048/169405147-bd68bafd-2138-45b3-9349-454de87685bb.png)
---------
- `figma` design 👇🏾 ;
![Screenshot (6)](https://user-images.githubusercontent.com/65298048/169405233-8f40395b-34a6-4161-862f-836fb68841ba.png)